sources/packages/web/src/views/institution/WithdrawalUpload.vue

```diff
@@ -0,0 +1,316 @@
+              ><template #body="slotProps">
+                <div
+                  class="alert alert-danger"
+                  v-if="slotProps.data.errors.length"
+                >
+                  <ul class="m-2">
+                    <li v-for="error in slotProps.data.errors" :key="error">
+                      {{ error }}
+                    </li>
+                  </ul>
+                </div>
+                <div
+                  class="alert alert-warning"
+                  v-if="slotProps.data.warnings.length"
+                >
+                  <ul class="m-2">
+                    <li
+                      v-for="warning in slotProps.data.warnings"
+                      :key="warning"
+                    >
+                      {{ warning.message }}
+                    </li>
+                  </ul>
+                </div>
+                <div
+                  class="alert alert-info"
+                  v-if="slotProps.data.infos.length"
+                >
+                  <ul class="m-2">
+                    <li v-for="info in slotProps.data.infos" :key="info">
+                      {{ info.message }}
+                    </li>
+                  </ul>
+                </div>
+              </template></Column
+            >
+          </DataTable>
+        </content-group>
+      </content-group>
+    </content-group>
+  </full-page-container>
+</template>
+
+<script lang="ts">
+import { ref, computed, defineComponent } from "vue";
+import {
+  DEFAULT_PAGE_LIMIT,
+  PAGINATION_LIST,
+  BannerTypes,
+  VForm,
+  InputFile,
+  ApiProcessError,
+} from "@/types";
+import { useFormatters, useSnackBar } from "@/composables";
+import { FileUploadProgressEventArgs } from "@/services/http/common/FileUploadProgressEvent";
+import { ApplicationService } from "@/services/ApplicationService";
+import { ApplicationBulkWithdrawal } from "@/types/contracts/institution/Application";
+import { APPLICATION_WITHDRAWAL_INVALID_TEXT_FILE_ERROR } from "@/constants";
+
+const ACCEPTED_FILE_TYPE = "text/plain";
+const MAX_APPLICATION_WITHDRAWAL_UPLOAD_SIZE = 4194304;
+
+export default defineComponent({
+  setup() {
+    const snackBar = useSnackBar();
+    const validationProcessing = ref(false);
+    const creationProcessing = ref(false);
+    const { dateOnlyLongString } = useFormatters();
+    // Only one will be used but the component allows multiple.
+    const withdrawalFiles = ref<InputFile[]>([]);
+    // Possible errors and warnings received upon file upload.
+    const validationResults = ref([] as ApplicationBulkWithdrawal[]);
+    const uploadForm = ref({} as VForm);
+    const uploadProgress = ref({} as FileUploadProgressEventArgs);
+    // Workaround to reset the file upload component to its original state.
+    // It is apparently a vuetify beta issue. It can be removed once there is a
+    // better way to force the component to reset its state.
+    const textFileUploadKey = ref(0);
```

Comment:
we are not on vuetify beta right? its vuetify 3? is that issue still there in 3?